test/collection_test.js

```diff
@@ -1443,6 +1443,32 @@ describe("Collection", () => {
           .then(res => res.data.map(r => r.title))
           .should.eventually.become(["art1"]);
       });
+
+      it("should support sub-object filtering", () => {
+        return articles
+          .list({
+            filters: { "article.complete": true },
```

Comment:
I don't think you can test the sub-object query with the entries of this test, since they don't have sub-objects in their fields.

You would require an objects like this one:
```js
{
  title: "art1",
  author: {
    name: "alice",
  }
}
```
And the query would be `{ filters: { "author.name": "alice" } }`